calicoctl get BGPConfiguration panics

<!--- Provide a general summary of the issue in the Title above -->
`calicoctl get BGPConfiguration` panics:

```
calicoctl get BGPConfiguration -o wide
panic: template: get:2:161: executing "get" at <.Spec.DefaultNodeASN...>: can't evaluate field DefaultNodeASNumber in type v2.BGPConfigurationSpec

goroutine 1 [running]:
github.com/projectcalico/calicoctl/calicoctl/commands.resourcePrinterTable.print(0x0, 0x0, 0x0, 0xc4203a7b01, 0x1ee3bc0, 0xc4201e8420, 0xc4202ae5c0, 0x1, 0x1, 0x1, ...)
	/home/bcreane/go/src/github.com/projectcalico/calicoctl/calicoctl/commands/printer.go:127 +0x461
github.com/projectcalico/calicoctl/calicoctl/commands.(*resourcePrinterTable).print(0xc42020abc0, 0x1ee3bc0, 0xc4201e8420, 0xc4202ae5c0, 0x1, 0x1, 0x0, 0x156ffbd)
	<autogenerated>:1 +0xa6
github.com/projectcalico/calicoctl/calicoctl/commands.Get(0xc4200d3080, 0x4, 0x4)
	/home/bcreane/go/src/github.com/projectcalico/calicoctl/calicoctl/commands/get.go:183 +0x3b1
main.main()
	/home/bcreane/go/src/github.com/projectcalico/calicoctl/calicoctl/calicoctl.go:83 +0x5f6
```

Interestingly, specifying `yaml` output format for the same resource works fine:

```
./calicoctl get BGPConfiguration -o yaml
apiVersion: projectcalico.org/v2
items:
- apiVersion: projectcalico.org/v2
  kind: BGPConfiguration
  metadata:
    creationTimestamp: null
    name: default
    resourceVersion: "42"
  spec:
    asNumber: 63400
    logSeverityScreen: info
    nodeToNodeMeshEnabled: true
kind: BGPConfigurationList
metadata:
  resourceVersion: "42"
```
## Expected Behavior
<!--- If you're describing a bug, tell us what should happen -->
Probably should not panic.
<!--- If you're suggesting a change/improvement, tell us how it should work -->

## Current Behavior
<!--- If describing a bug, tell us what happens instead of the expected behavior -->
Panics.
<!--- If suggesting a change/improvement, explain the difference from current behavior -->

## Possible Solution
<!--- Not obligatory, but suggest a fix/reason for the bug, -->
<!--- or ideas how to implement the addition or change -->

## Steps to Reproduce (for bugs)
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
1. Setup an etcdv3 datastore with calico v3.0.0-alpha1.
2. Create a bgp resource, e.g.: 
```
apiVersion: projectcalico.org/v2
kind: BGPConfiguration
metadata:
  name: default
spec:
  logSeverityScreen: info
  nodeToNodeMeshEnabled: true
  asNumber: 63400
```
3. `calicoctl get BGPConfiguration` # panics
4. `calicoctl get BGPConfiguration -o yaml` # works

## Context
<!--- How has this issue affected you? What are you trying to accomplish? -->
<!--- Providing context helps us come up with a solution that is most useful in the real world -->

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* calicoctl version (e.g. `calicoctl version`):
```
Client Version:    v2.0.0-alpha1
Build date:        2017-10-31T22:48:12+0000
Git commit:        2b700a3a
Cluster Version:   v3.0.0-alpha1-1-gcfce279
Cluster Type:      unknown
```
* calico/node version (e.g. `docker run --rm calico/node versions`)
```
confd v1.0.0-alpha1-rc3
BIRD version 1.6.3
libnetwork-plugin: v1.1.0
calico-bgp-daemon: v0.2.1

Felix:
2017-11-01 00:21:47.662 [INFO][963] logutils.go 83: Early screen log level set to info
Version:            3.0.0-alpha1-rc1
Full git commit ID: 0aba6bc56f0a8990795e2ae491a13bd22b0c4683
Build date:         2017-10-25T01:53:47+0000
```
* Orchestrator type and version (e.g. kubernetes, mesos, rkt):
* Operating System and version:
* Link to your project (optional):

--- a/calicoctl/resourcemgr/bgpconfig.go
+++ b/calicoctl/resourcemgr/bgpconfig.go
@@ -28,13 +28,13 @@ func init() {
 		api.NewBGPConfigurationList(),
 		false,
 		[]string{"bgpconfiguration", "bgpconfigurations", "bgpconfig", "bgpconfigs"},
-		[]string{"NAME", "LOGSEVERITY", "MESHENABLED", "DEFAULTASN"},
-		[]string{"NAME", "LOGSEVERITY", "MESHENABLED", "DEFAULTASN"},
+		[]string{"NAME", "LOGSEVERITY", "MESHENABLED", "ASNUMBER"},
+		[]string{"NAME", "LOGSEVERITY", "MESHENABLED", "ASNUMBER"},
 		map[string]string{
 			"NAME":        "{{.ObjectMeta.Name}}",
 			"LOGSEVERITY": "{{.Spec.LogSeverityScreen}}",
 			"MESHENABLED": "{{if .Spec.NodeToNodeMeshEnabled}}{{.Spec.NodeToNodeMeshEnabled}}{{ else }}-{{ end }}",
-			"DEFAULTASN":  "{{if .Spec.DefaultNodeASNumber}}{{.Spec.DefaultNodeASNumber}}{{ else }}-{{ end }}",
+			"ASNUMBER":  "{{if .Spec.ASNumber}}{{.Spec.ASNumber}}{{ else }}-{{ end }}",
 		},
 		func(ctx context.Context, client client.Interface, resource ResourceObject) (ResourceObject, error) {
 			r := resource.(*api.BGPConfiguration)

